src/gui/MainWindow.cpp

```diff
@@ -308,12 +308,13 @@ void MainWindow::finalize()
 					SLOT( exportProjectTracks() ),
 					Qt::CTRL + Qt::SHIFT + Qt::Key_E );
 
-	// temporarily disabled broken MIDI export				
-	/*project_menu->addAction( embed::getIconPixmap( "midi_file" ),
+#ifdef LMMS_HAVE_FLUIDSYNTH
```

Comment:
@tresf @softrabbit How do you think a new plugin just for MIDI-out like FL Studio?

I have no good idea about that.